enterprise/backend/test/metabase_enterprise/content_translation/dictionary_test.clj

```diff
@@ -0,0 +1,173 @@
+(ns metabase-enterprise.content-translation.dictionary-test
+  "Tests for content translation dictionary import logic."
+  (:require
+   [clojure.test :refer :all]
+   [metabase-enterprise.content-translation.dictionary :as dictionary]
+   [metabase-enterprise.content-translation.utils :as ct-utils]
+   [metabase.test :as mt]
+   [metabase.util.log :as log]
+   [toucan2.core :as t2]))
+
+(set! *warn-on-reflection* true)
+
+(defn- count-translations
+  "Count the number of translations in the database."
+  []
+  (count (t2/select :model/ContentTranslation)))
+
+(defn- get-translations
+  "Get all translations from the database."
+  []
+  (t2/select :model/ContentTranslation))
```

Comment:
should this be ordered? alphabetized or something? just thinking that we might want to give these to people in a consistent order